scripts/constants.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+# Use lower_case variables in the scripts and UPPER_CASE variables for override
+# Use the constants.sh for env overrides
+
+REPO_PATH=$(
```

Comment:
Since we're sourcing `constants.sh` from multiple repos, I'm slightly in favor of have the pattern for this variable reference the name of the repo itself. It doesn't seem unreasonable for somebody to see `REPO_NAME` here, and go to update the corresponding variable name in `teleporter/constants.sh` to  match, causing a variable conflict.

Reply:
Ohhh yes good points, thanks